config/config.php

```diff
@@ -225,7 +225,7 @@
 			],
 		],
 		'13' => [
-			'15' => [
+			'50' => [
```

Comment:
This basically overwrites the next element. As we are using array keys, the key `50` would be there twice and PHP then only uses one of them. Let me fix this.